src/index.ts

```diff
@@ -1,21 +1,22 @@
 import * as getRawBody from 'raw-body';
 import * as Koa from 'koa';
 import * as Router from 'koa-router';
-import MockDidCache from '../tests/mocks/MockDidCache'; // TODO: Replace once real implementation comes in.
+import { DidCache } from './DidCache';
 import Observer from './Observer';
 import RequestHandler from './RequestHandler';
 import Rooter from './Rooter';
 import { BlockchainClient } from './Blockchain';
 import { CasClient } from './Cas';
 import { Config, ConfigKey } from './Config';
 import { toHttpStatus, Response } from './Response';
+import { didDocumentUpdate, didDocumentCreate } from '../tests/mocks/MockDidDocGen';
```

Comment:
Avoid using abbreviation. Not sure what Gen stands for, but perhaps MockDidDocumentGenerator?

Reply:
Changed MockDidDocGen.ts => MockDidDocumentGenerator.ts